data/101/993/581/101993581.geojson

```diff
@@ -29,10 +29,11 @@
         "Playas de Rosarito"
     ],
     "name:eng_x_preferred":[
-        "Rosario"
+        "Playas de Rosarito"
```

Comment:
For the preferred name let's just say `Rosarito`, and have the longer `Playas de Rosarito` as a variant instead, please.

Reply:
(Also update WOF name)

Reply:
Updated in 4c37574.